src/async-components/views/dialogs/secretstorage/CreateSecretStorageDialog.js

```diff
@@ -225,13 +225,21 @@ export default class CreateSecretStorageDialog extends React.PureComponent {
         const { force } = this.props;
 
         try {
-            await cli.bootstrapSecretStorage({
-                setupNewSecretStorage: force,
-                authUploadDeviceSigningKeys: this._doBootstrapUIAuth,
-                createSecretStorageKey: async () => this._keyInfo,
-                keyBackupInfo: this.state.backupInfo,
-                setupNewKeyBackup: force || !this.state.backupInfo && this.state.useKeyBackup,
-            });
+            if (!force) {
```

Comment:
Nit: Could you flip these block around so this can be `if (force)` instead? I think it would easier to read without the mental negation.

Reply:
sure